presto-main/src/main/java/io/prestosql/logging/aspects/SqlCallFlowLoggingAspect.java

```diff
@@ -0,0 +1,163 @@
+     * @return the same object that is returned by the method that is wrapped in this advice
+     * @throws Throwable the same exception that is thrown by the method that is wrapped
+     */
+    @Around(START_METHOD)
+    public Object startFlux(final ProceedingJoinPoint point)
+            throws Throwable
+    {
+        final long threadId = Thread.currentThread().getId();
+
+        threadIdToStep.put(threadId, 0);
+        threadIdToDebugLogId.compute(
+                threadId, (key, value) -> UUID.randomUUID().getMostSignificantBits());
+
+        return printDebugLogForMethod(point, threadId);
+    }
+
+    /**
+     * This method just retrieves the thread id and delegate the processing to method inside {@link
+     * BaseAspect}
+     *
+     * <p>It will delegate the processing to {@link
+     * BaseAspect#printDebugLogForMethod(ProceedingJoinPoint, long)} that will be responsible to
+     * insert inside the log information about method to be executed.
+     *
+     * @param point an object o Aspectj library that represents the method that ShannonDB needs to
+     * obtain information inside logs.
+     * @return the same object that is returned by the method that is wrapped in this advice
+     * @throws Throwable the same exception that is thrown by the method that is wrapped
+     */
+    @Around(WHITE_AND_BLACK_LIST)
+    public Object around(final ProceedingJoinPoint point)
+            throws Throwable
+    {
+        final long threadId = Thread.currentThread().getId();
+
+        return printDebugLogForMethod(point, threadId);
+    }
+
+    /**
+     * This method is executed when the Thread starts execution when finishes the command of create
+     * schema inside the worker.
+     *
+     * <p>It is necessary to remove the step and debug log id variable for each thread that already
+     * executed the flux. It will first to delegate the processing to {@link
+     * BaseAspect#printDebugLogForMethod(ProceedingJoinPoint, long)} that will be responsible to
+     * insert inside the log information about method to be executed than it will reset information
+     * about step and debug log id inside maps.
+     *
+     * @param point an object o Aspectj library that represents the method that ShannonDB needs to
+     * obtain information inside logs.
+     * @return the same object that is returned by the method that is wrapped in this advice
+     * @throws Throwable the same exception that is thrown by the method that is wrapped
+     */
+    @Around(FINISH_METHOD)
+    public Object finishFlux(final ProceedingJoinPoint point)
+            throws Throwable
+    {
+        final long threadId = Thread.currentThread().getId();
+
+        final Object resultFromMethod = printDebugLogForMethod(point, threadId);
+
+        threadIdToStep.remove(threadId);
+        threadIdToDebugLogId.remove(threadId);
+
+        return resultFromMethod;
+    }
+
+    /**
+     * Defines a name for a general flux inside ShannonDB
```

Comment:
Do not use the word "flux". Always use flow. Please, search for all the use of the word "flux" and change it please

Reply:
fixed